CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to this repository
+
+## Learn more about Upstream
+
+Looking to get familiar with the Upstream app and the Radicle ecosystem:
+
+* [DEVELOPMENT.md][dmd]
+* [Discord][dc]
+* [radicle.xyz][ra]
+* [Radicle documentation][rd]
+
+## Your first bugfix
+
+If you have an issue you know how to fix or see an [open issue][oi] you'd like
+to fix, we are keen to receive your contributions.
+
+We are currently accepting [PR's through GitHub][pr] while we make our own
+infrastructure more robust. If it's your first time using GitHub Pull Requests,
+learn more about it [here][ghf].
+
+We also accept contributions through [Upstream Patches][up], our first version
+of Pull Requests in Upstream. Learn more about the contribution flow [here][cbf].
+For us to see you've contributed a Patch we need to [add your Device ID as a
+remote][ar] to our repo, you can either add it to the [Remotes to track
+issue][rtr] or post it in [#upstream on Discord][dc-up].
+
+## Documentation
+
+We're writing documentation as we are developing new features if you find
+something that is confusing or not covered at all, feel free to [open a bug][ob]
+or [contribute][cd]
```

Comment:
```suggestion
We're writing documentation as we are developing new features, if you find
something that is confusing or not covered at all, feel free to [report a bug][ob]
or [contribute][cd].
```

Reply:
Turned it into two sentences in https://github.com/radicle-dev/radicle-upstream/pull/1935/commits/7d3d5751f26acf6337ca4cef8c77f3968cc4ba61